packages/fxa-auth-server/lib/senders/index.js

```diff
@@ -115,6 +115,21 @@ module.exports = (log, config, error, bounces, translator, oauthdb, sender) => {
           );
         });
       },
+      sendVerifyShortCode: function(emails, account, opts) {
+        // This function defers from `sendVerifyCode` since it sends a code that
+        // is expected to be entered into a input field rather than a link to verify
+        // the account.
```

Comment:
With this comment in mind, and to keep the nomenclature consistent at every level, should we call the partial/templates `verify_short_code.*` and the method above `verifyShortCodeEmail`?

Reply:
Also typo, just noticed: `s/defers/differs/`